### DISCUSSION TITLE
感叹一下米家美工是真的不要钱

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#794] [General]
一个网页活动阿贝多变的蝴蝶比风起地树叶帧率都高
<img width="1260" alt="image" src="https://user-images.githubusercontent.com/9006264/164598622-3e21900a-a0bb-4a24-9fc2-06a1660c2304.png">

**Comment by Xm798** [comment]
米家……我以为是那个米家
